curl_cffi/requests/impersonate.py

```diff
@@ -1,12 +1,63 @@
-from dataclasses import dataclass
-from typing import List, Literal, Optional, TypedDict
 import warnings
+from dataclasses import dataclass
 from enum import Enum
+from typing import List, Literal, Optional, TypedDict
 
-from ..const import CurlSslVersion, CurlOpt
+from ..const import CurlOpt, CurlSslVersion
 
+BrowserTypeLiteral = Literal[
+    # Edge
+    "edge99",
+    "edge101",
+    # Chrome
+    "chrome99",
+    "chrome100",
+    "chrome101",
+    "chrome104",
+    "chrome107",
+    "chrome110",
+    "chrome116",
+    "chrome119",
+    "chrome120",
+    "chrome123",
+    "chrome124",
+    "chrome99_android",
+    # Safari
+    "safari15_3",
+    "safari15_5",
+    "safari17_0",
+    "safari17_2_ios",
+    # alias
+    "chrome",
+    "edge",
+    "safari",
+    "safari_ios",
+    "chrome_android",
+]
 
-class BrowserType(str, Enum):
+DefaultChrome = "chrome124"
+DefaultEdge = "edge101"
+DefaultSafari = "safari17_0"
+DefaultSafariPhone = "safari17_2_ios"
+DefaultChromePhone = "chrome99_android"
```

Comment:
For constants, `DEFAULT_CHROME` is preferred over `DefaultChrome`.

Reply:
`DefaultChromePhone` should be `DEFAULT_CHROME_ANDROID`. All browsers on iOS uses the webkit core, which means they use the webkit fingerprints.